js/api.js

```diff
@@ -0,0 +1,48 @@
+import { FIREBASE_URL } from './constants.js';
+
+export const apiGetImages = async () => {
+    try {
+        const response = await fetch(FIREBASE_URL + 'images.json');
+        return await response.json();
+    } catch (error) {
+        console.log(error);
+        return null;
+    }
+};
+
+export const ApiSetImage = async (imageData) => {
```

Comment:
```suggestion
export const apiSetImage = async (imageData) => {
```
Z WIELKIEJ litery piszemy tylko klasy!!